scalafix/input/src/main/scala/fix/ClientFetchTests.scala

```diff
@@ -19,31 +23,30 @@ object ClientFetchTests {
       this.fetch[A](req)(f)
 
     final def doFetchR2[B](req: Request[G], f: Response[G] => G[B])(implicit G: Sync[G]): G[B] =
-      fetch[B](req) { f }
+      fetch[B](req)(f)
```

Comment:
oh, I guess this change is due to scalafmt rules. And it would be totally fine for production code.
But actually it will produce a different syntactic tree in scalafix so this particular case and some other below lose any sense.

We should be able to preserve these input and output files from scalafmt since they are intended to handle different ways of formatting and structuring that some other uses may have in their projects.

Reply:
I think we aren't free to assume that all people who use http4s in their project have the same scalafix rules as well.

Reply:
Oh, that's a very good point.  I'll turn off formatting and revert the inputs and outputs.  And the license headers are more pain than they're worth, here.

We can keep the usual checks in the rules and tests projects, I think.